PNlib/PC.mo

```diff
@@ -13,9 +13,9 @@ model PC "Continuous Place"
   parameter Integer enablingType=1
     "resolution type of actual conflict (type-1-conflict)" annotation(Dialog(enable = true, group = "Enabling"), choices(choice=1
         "Priority", choice=2 "Probability", __Dymola_radioButtons=true));
-  parameter Real enablingPrioIn[nIn]=if nIn>1 then linspace(1,nIn,nIn) else fill(1, nIn)
+  parameter Integer enablingPrioIn[nIn]=if nIn>1 then integer(linspace(1,nIn,nIn)) else fill(1, nIn)
```

Comment:
Probably, it would be better to just write `parameter Integer enablingPrioIn[nIn]=if nIn>1 then 1:nIn else fill(1, nIn)`.